src/devices/govee.c

```diff
@@ -198,12 +198,21 @@ static int govee_decode(r_device *decoder, bitbuffer_t *bitbuffer)
     event &= 0x0FFF;
 
     char const *event_str;
+    int wet = -1;
     // Figure out what event was triggered
     if (event == 0xafa) {
         event_str = "Button Press";
+        // The H5054 water sensor does not send a message when it transitions from wet to dry nor does it have a
+        // dedicated message to indicate that it is not wet. However, the sensor only sends a "button press" message if
+        // the button is pressed while the device is dry (no button press message is sent if the button is pressed while
+        // the sensor is wet). Since we know the sensor is dry when a "button press" message is received, "detect_wet:0"
+        // is included in the output when the button is pressed as a workaround to allow the user to transition the
+        // device back to the dry state.
+        wet = 0;
```

Comment:
seems like a merge error.  not sure what's going on.

Reply:
Same as above